### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by Moore2021**
Beginning process

### CODE REVIEW

**Review by klerikdust — APPROVED** [review]

File: src/annie.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const customConfig = require(`./config/customConfig.js`)
 const config = require(`./config/global`)
 const commandsLoader = require(`./commands/loader`)
 const Database = require(`./libs/database`)
-const localizer = require(`./libs/localizer`)
+const {localizer, getTargetLocales} = require(`./libs/localizer`)
```
Comment (klerikdust) [inline src/annie.js:7]: Does `localizer` function exist inside /libs/localizer? If I read it correctly there was only getTargetLocales() being exported

Comment (Moore2021) [reply src/annie.js]: that is correct, that's a ghost variable, I just pushed a small commit to get rid of it.

File: tests/LocalesTests/integrity.js
Comment (klerikdust) [inline tests/LocalesTests/integrity.js:1]: Why this unit-test is so freaking verbose😂
Try to import the existing `localizer` function(s) to here and write the test based on the expected output from it.

Comment (klerikdust) [inline tests/LocalesTests/integrity.js:1]: I also want to say, go use the the `chai` framework .. as I already use `chai` in other unit tests.

Comment (Moore2021) [reply tests/LocalesTests/integrity.js:1]: the other function isn't used, it's two separate functions

**Review by klerikdust — CHANGES_REQUESTED** [review]
Pointed out some revisions

File: tests/LocalesTests/integrity.js
Comment (klerikdust) [reply tests/LocalesTests/integrity.js:1]: although this is not important lol.. just consistency I would say

Comment (klerikdust) [reply tests/LocalesTests/integrity.js:1]: aah the node version .. i see

Comment (klerikdust) [reply tests/LocalesTests/integrity.js:1]: scratch this part then, not quite important

**Pull request merged**